ui/desktop/src/components/ViewRecipeModal.tsx

```diff
@@ -73,7 +71,8 @@ export default function ViewRecipeModal({ isOpen, onClose, config }: ViewRecipeM
       setInstructions(config.instructions || '');
       setPrompt(config.prompt || '');
       setActivities(config.activities || []);
-      setParameters(parseParametersFromInstructions(config.instructions || ''));
+      // Use the actual parameters from the config, not parsed from instructions
+      setParameters(config.parameters || []);
```

Comment:
remove this comment, it explains the code change, but is not needed for the reader